src/main/webapp/usermap.jsp

```diff
@@ -15,7 +15,7 @@
     <h1 class="caption">Who is using TEAMMATES?</h1>
     <div id="contentHolder">
         <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE;"></div>
-        <p id="lastUpdate" class="lastUpdate">Last updated: 30 November 2016</p>
+        <p id="lastUpdate" class="lastUpdate">Last updated: <span id="lastUpdateDate" class="lastUpdate"></span></p>
```

Comment:
If i remove the class then the **Last Updated** phrase and the date will be in different style.But i will remove the id on the < p > tag.Is that ok?